runbotics/runbotics-desktop/src/action/sap/sap.action-handler.ts

```diff
@@ -91,35 +132,81 @@ export default class SapActionHandler extends StatefulActionHandler {
         return array;
     }
 
+    /**
+     *  @alias focus 
+     *  @name SAP: Focus
+     *  @description Selects an element and "highlights" it.
+     *  @param target
+     *  @example focus('wnd[0]/example/example')
+     */
     async focus(input: SapTypes.SAPFocusActionInput): Promise<SapTypes.SAPFocusActionOutput> {
         this.isApplicationOpen();
         this.session.FindById(input.target).setFocus();
         return {};
     }
 
+    /**
+     *  @alias disconnect 
+     *  @name SAP: Disconnect
+     *  @description Disconnects from instance on which the bot is currently logged in.
+     */
     async disconnect() {
         await this.session?.Parent.CloseConnection();
         this.session = null;
     }
 
+    /**
+     *  @alias sendVKey 
+     *  @name SAP: Send Virtual Key
+     *  @description Indicates keyboard shortcut to be clicked by bot.
+     *  @param virtualKey
+     *  @example sendVKey(Enter)
+     *  @see SendVKeyMapper - check out more examples
```

Comment:
whats `SendVKeyMapper`?

Reply:
> whats `SendVKeyMapper`?

`SendVKeyMapper` is a keymap file that contains the keyboard shortcuts we support in SAP

`@see SendVKeyMapper` - file reference so user can see more examples